### PR TITLE
Create a new page for yaml editor

### LINKED ISSUE #5172: Create a new page for yaml editor
# Motivation
Currently, OpenPAI webportal does not support submit yaml directly. User's yaml config will be sequelized and set some default values.

# Goal
Create a new page for user to submit a protocol yaml config directly. And provide a editor to edit the config.

# Test cases
# Known issues
# Future work

### PR DESCRIPTION
**Pull request by SwordFaith**
Add a new job config editor page.

Make it can responsive to various resolution.

close #5172

### CODE REVIEW

File: src/webportal/src/app/job-submission/components/yamledit-topbar/export-config.jsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
+/*
```
Comment (debuggy) [inline src/webportal/src/app/job-submission/components/yamledit-topbar/export-config.jsx:1]: add lisence header following https://docs.opensource.microsoft.com/content/releasing/copyright-headers.html?q=header

File: src/webportal/src/app/job-submission/components/yamledit-topbar/yamledit-topbar.jsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import { ExportConfig } from './export-config';
+import { ImportConfig } from './import-config';
+
+export const YamlEditTopBar = ({ protocolYaml, onChange }) => {
+  return (
+    <Stack horizontal horizontalAlign='space-between' padding='0 m'>
+      <Stack horizontal gap='m' verticalAlign='baseline'>
+        <StackItem>
+          <Text variant='xLarge' styles={{ root: { fontWeight: 'semibold' } }}>
+            Protocol Yaml Edit Page
```
Comment (debuggy) [inline src/webportal/src/app/job-submission/components/yamledit-topbar/yamledit-topbar.jsx:15]: Align with `Config Editor`?

File: src/webportal/src/app/job-submission/yaml-edit-page.jsx [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+    } else {
+      setValidStatus({
+        message: 'Success: Validation completed successfully',
+        barType: MessageBarType.success,
+      });
+    }
+  }, [protocolYaml]);
+
+  const _submitJob = async event => {
+    console.log('Submit job triggered');
```
Comment (debuggy) [inline src/webportal/src/app/job-submission/yaml-edit-page.jsx:56]: delete console.log

File: src/webportal/src/app/job-submission/components/yamledit-topbar/export-config.jsx (hunk 1 above)
Comment (SwordFaith) [reply src/webportal/src/app/job-submission/components/yamledit-topbar/export-config.jsx]: Added in new commit

File: src/webportal/src/app/job-submission/yaml-edit-page.jsx (hunk 3 above)
Comment (SwordFaith) [reply src/webportal/src/app/job-submission/yaml-edit-page.jsx]: Removed in new commit

File: src/webportal/src/app/job-submission/yaml-edit-page.jsx [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+              theme: 'vs',
+              onChange: debounce(setProtocolYaml, 100),
+              value: protocolYaml,
+              options: {
+                language: 'yaml',
+                wordWrap: 'on',
+              },
+            }}
+            onChange={(newValue, e) => {
+              console.log('onChange', newValue, e);
```
Comment (debuggy) [inline src/webportal/src/app/job-submission/yaml-edit-page.jsx:101]: does this onChange conflict with monacoProps?

Comment (SwordFaith) [reply src/webportal/src/app/job-submission/yaml-edit-page.jsx:101]: This onChange seems no match props, so will have no effect

**Comment by debuggy** [comment]
LGTM :shipit:

**Review by debuggy — APPROVED** [review]

**Pull request merged**